content/concepts/discovery-routing/mDNS.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "mDNS"
+description: "mDNS uses a multicast system of DNS records over a local network to enable peer discovery."
+weight: 224
+---
+
+## What is mDNS?
+
+mDNS, or multicast Domain Name System, is a way for nodes to use a multicast system
+of DNS records over a local network to discover and communicate with nodes. Nodes
+broadcast topics they're interested in instead of querying a central name server.
+The discovery, however, is limited to the peers in the local network. mDNS is commonly
+used on home networks to allow devices such as computers, printers, and smart TVs to
+find each other and connect. It uses a protocol called multicast to broadcast messages
+on the network, allowing devices to discover each other and exchange information.
+
+## mDNS in libp2p
+
+In libp2p, mDNS is used for peer discovery, allowing peers to find and
+communicate with each other on the same local network without any prior configuration.
+This is achieved through multicast DNS (mDNS) records, which are sent to all nodes on the
+local network.
+
+To initiate peer discovery, a peer sends a query to all other peers on the network using
```

Comment:
I would suggest getting rid of everything below here until the `Example` section. That one is kind of cool to see. Maybe add a bit more context for the example since we've removed this.